.github/workflows/backport.yml

```
@@ -0,0 +1,27 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    # Only react to merged PRs for security reasons.
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport')
+        )
+      )
+    steps:
+      - uses: sourcegraph/backport@v2
+        with:
+          github_token: ${{ secrets.BACKPORT_GITHUB_TOKEN }}
+          label_pattern: '^backport (?<base>jetbrains-v\d+\.\d+\.x)$'
```

```suggestion
          label_pattern: '^backport (?<base>jb-v\d+\.\d+\.x)$'
```

WOOPs my bad, the branch should be `jb-vX.YZ.x` so let's match this with the label. I need to align with Dominic on what we think the label should look like, can we freely change the pattern once this merged or will it require backport library updates as well?

---

You can freely change the label_pattern here without needing any changes to the backport library :)

---

woooo ok ty!

---

The only thing is that it needs is the named capture group for `base`